server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/WebSocketContract.scala

```diff
@@ -362,6 +362,37 @@ trait WebSocketContract {
                    |}""".stripMargin)
   }
 
+  @Test
+  @Disabled("Not implemented yet")
+  def pushEnableRequestsShouldBeProcessed(server: GuiceJamesServer): Unit = {
+    val response: Either[String, String] =
+      authenticatedRequest(server)
+        .response(asWebSocket[Identity, String] {
+          ws =>
+            ws.send(WebSocketFrame.text(
+              """{
+                |  "@type": "WebSocketPushEnable",
+                |  "dataTypes": ["Mailbox", "Email"]
+                |}""".stripMargin))
+
+            ws.receive()
+              .map { case t: Text => t.payload }
+        })
+        .send(backend)
+        .body
+
```

Comment:
We may need an extra operation (like receiving a mail) to make the state notification happen